docker/vttestserver/Dockerfile.mysql57

```diff
@@ -55,4 +55,7 @@ COPY --from=builder --chown=vitess:vitess /vt/install /vt
 VOLUME /vt/vtdataroot
 USER vitess
 
-CMD /vt/bin/vttestserver -port $PORT -keyspaces $KEYSPACES -num_shards $NUM_SHARDS -mysql_bind_host ${MYSQL_BIND_HOST:-127.0.0.1} -mysql_server_version ${MYSQL_SERVER_VERSION:-5.7.9-vitess}
+COPY docker/vttestserver/setupVschemaFolder.sh /vt/setupVschemaFolder.sh
+COPY docker/vttestserver/runMysql57.sh /vt/run.sh
```

Comment:
nit: can you rename the files to use underscores / dashes instead of camelCase?
Suggestions:
setupVschemaFolder.sh -> setup-vschema.sh 
runMysql57.sh -> run-mysql57.sh
runMysql80.sh -> run-mysql80.sh

Reply:
Alright will do that

Reply:
done